internal/worker/json.go

```diff
@@ -736,3 +736,16 @@ type AWSEC2CopyJobResult struct {
 	Ami    string `json:"ami"`
 	Region string `json:"region"`
 }
+
+// ImageBuilderManifestJob generates a manifest from a build request using
+// image-builder-cli. Includes resolving all content types.
+type ImageBuilderManifestJob struct {
+	// Arguments to the image-builder command line
+	Args ImageBuilderArgs
```

Comment:
I'm sure that you are aware of this, but with this change, the `blueprint.Blueprint` becomes part of the Worker API, so any changes to it will affect the worker. It shouldn't be a problem, since the package is intended for user-facing serialization.